### DISCUSSION TITLE
Welcome to energy-period-selector-plus Discussions!

### DISCUSSION THREAD

**Discussion by flixlix** [flixlix/energy-period-selector-plus#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mhuiskes** [comment]
Hi Flixlix,

Thanks for creating energy-period-selector-plus! It's working great!

Two questions:

- Would it be possible to support two tariffs for the grid? The regular energy card allows for multiple tariffs. The energy meter provides 4 sensors: Energy production Tarif 1, Energy production Tarif 2, Energy consumption Tarif 1 and Energy consumption Tarif 2. 

- Some times the card remains empty with a message 'loading'. This happens both in the web screen as well as the mobile app for home assistant. I haven't figured out how to consistently reproduce it..

Cheers,
Mike
